src/parser/addressRepresentationConverters.ts

```diff
@@ -122,6 +123,26 @@ export const simpleCellAddressFromString = (sheetMapping: SheetMappingFn, string
   return simpleCellAddress(sheet, col, row)
 }
 
+export const simpleCellRangeFromString = (sheetMapping: SheetMappingFn, stringAddress: string, sheetContext: number): Maybe<SimpleCellRange> => {
+  const split = stringAddress.split(':')
```

Comment:
Might be out of scope, but would be cool to have this symbol in one place.

```suggestion
  const split = stringAddress.split(RANGE_OPERATOR)
```
Should be the same as in lexer if export it from LexerConfig: https://github.com/handsontable/hyperformula/blob/6ec629b16035174d3f63e418d05cd0e98e682eb3/src/parser/LexerConfig.ts#L65



BTW And similar for the whole file wherever we use `$` symbol
```diff
-if (result[5] === '$') {
+if (result[5] === ABSOLUTE_OPERATOR) {
```